herddb-thirdparty/openjpa-test/src/main/resources/META-INF/persistence.xml

```diff
@@ -22,17 +22,29 @@
              xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
   <persistence-unit name="hdb_ds" transaction-type="RESOURCE_LOCAL">
     <class>test.entity.User</class>
+    <class>test.entity.Address</class>
     <exclude-unlisted-classes>true</exclude-unlisted-classes>
     <properties>
+      <property name="openjpa.Log" value="SQL=TRACE"/>
       <property name="javax.persistence.schema-generation.database.action" value="create" />
-       <property name="openjpa.jdbc.DBDictionary" value="herddb.openjpa.DBDictionary"/>
+<!--      <property name="openjpa.jdbc.SchemaFactory" 
+          value="native(foreignKeys=true)" />
+-->      <property name="openjpa.jdbc.MappingDefaults" 
+          value="ForeignKeyDeleteAction=cascade, JoinForeignKeyDeleteAction=cascade"/>
+      <property name="openjpa.jdbc.DBDictionary" value="herddb.openjpa.DBDictionary"/>
     </properties>
   </persistence-unit>
   
   <persistence-unit name="hdb_jdbc" transaction-type="RESOURCE_LOCAL">
     <class>test.entity.User</class>
+    <class>test.entity.Address</class>
     <exclude-unlisted-classes>true</exclude-unlisted-classes>
     <properties>
+      <property name="openjpa.Log" value="SQL=TRACE"/>      
+<!--      <property name="openjpa.jdbc.SchemaFactory" 
+          value="native(foreignKeys=true)" />
```

Comment:
"native" creates real FKs but it does not work with "demiters", I will work on OpenJPA side in order to fix this problem